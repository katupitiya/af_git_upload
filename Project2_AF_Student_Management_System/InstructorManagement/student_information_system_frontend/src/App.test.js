import React from 'react';
import Add from './routers/Instructor/AddAssignmentDetails';
import { create } from 'react-test-renderer'

describe('My first snapshot test',()=>{
  test('testing app button', () => {
    let tree = create(<Add />)
    expect(tree.toJSON()).toMatchSnapshot();
  })
})