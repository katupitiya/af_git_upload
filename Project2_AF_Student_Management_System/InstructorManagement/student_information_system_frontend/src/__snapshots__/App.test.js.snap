// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My first snapshot test testing app button 1`] = `
<div>
  <div
    className="assignment_form"
    style={
      Object {
        "marginTop": "-140px",
      }
    }
  >
    <form
      onSubmit={[Function]}
    >
      <div
        className="main-form"
      >
        <label>
          Subject Name:
        </label>
        <select
          className="form-input"
          id="name"
          name="SubjectName"
          onChange={[Function]}
          value=""
        >
          <option
            value="select Subject"
          >
            Select Subject
          </option>
          <option
            value="Application Framework"
          >
            Application Framework
          </option>
          <option
            value="Distributed System"
          >
            Distributed System
          </option>
        </select>
      </div>
      <div
        className="main-form"
      >
        <label>
          Assignment Name:
        </label>
        <input
          className="form-input"
          name="AssignmentName"
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
      <div
        className="main-form"
      >
        <label>
          Due Date:
        </label>
        <input
          className="form-input"
          name="date"
          onChange={[Function]}
          type="date"
          value=""
        />
      </div>
      <div
        className="main-form"
      >
        <button
          className="form-btn"
        >
          Submitt Assignment
        </button>
      </div>
    </form>
  </div>
</div>
`;
